src/Silverback.Integration.Kafka/Messaging/Messages/KafkaMessageHeaders.cs

```diff
@@ -23,5 +23,35 @@ public static class KafkaMessageHeaders
         ///     The header that will be filled with the timestamp of the message consumed from Kafka.
         /// </summary>
         public const string TimestampKey = "x-kafka-message-timestamp";
+
+        /// <summary>
+        ///     This will be set by the Move error policy and will contain the group ID of the consumer who processed the
```

Comment:
* Reference the error policy in the summary with `<see cref=`, e.g. `/// This will be set by the <see cref="MoveMessageErrorPolicy" /> ...`.
* Instead of calling it `the failed message`, call it `the message that failed to be processed` (the same expression is used already in the rest of the documentation, e.g. https://github.com/BEagle1984/silverback/blob/master/src/Silverback.Integration/Messaging/Inbound/ErrorHandling/MoveMessageErrorPolicy.cs#L17).

Reply:
Summaries fixed.